### PR TITLE
Work done.

### PR DESCRIPTION
**Pull request by astrohlicpu**
Hi,
Your scenario was really nicely made and included some challenges!
Please look over my work and let me know if you want any changes.

### CODE REVIEW

File: TherapyClinicDB/1 Tables.sql [hunk 1]
```diff
@@ -0,0 +1,24 @@
+use TherapyClinicDB
+go 
+
+drop table if exists ClientRecord
+go
+create table dbo.ClientRecord(
+    ClientRecordId int primary key identity,
+    FirstName varchar(50) not null constraint ck_ClientRecord_FirstName_cannot_be_blank check(FirstName <> ''),
+    LastName varchar(30) not null constraint ck_ClientRecord_LastName_cannot_be_blank check(LastName <> ''),
+    DOB date not null constraint ck_ClientRecord_DOB_needs_to_be_more_then_8_years_old check(datediff(year,DOB,getdate()) >=8 ),
```
Comment (AvrumiG) [inline TherapyClinicDB/1 Tables.sql]: The date check constraint doesn't ensure that the client was 8yr old at the begining of the therapy, rather that he is older then 8 as of the current date.

File: TherapyClinicDB/1 Tables.sql [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    DateStarted date not null,
+    DateFinished date null,
+    Frequency varchar(12) not null constraint ck_ClientRecord_Frequency_must_be_in_weekly_bi_weekly_or_twice_a_week check(Frequency in ('weekly', 'bi-weekly', 'twice a week')),
+    TherapyReason varchar(35) not null constraint ck_ClientRecord_TherapyReason_cannot_be_Borderline_Personality_Disorder_or_Schizophrenia check(TherapyReason not in ('Schizophrenia','Borderline Personality Disorder','BPD')),
```
Comment (AvrumiG) [inline TherapyClinicDB/1 Tables.sql]: You haven't blocked blanks in the TherapyReason field. 
(Although the spec didn't clearly state that a value is mandatory, I would assume that it is)

File: TherapyClinicDB/1 Tables.sql [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    PrevDiagnoses varchar(100) not null,
+    Progress tinyint not null constraint ck_ClientRecord_progress_can_only_be_between_1_and_5 check (Progress between 1 and 5),
+    ProgressDesc as case Progress when 1 then 'Very worsened'
+                        when 2 then 'Worsened'
+                        when 3 then 'Stayed the same'
+                        when 4 then 'Improved'
+                        when 5 then 'Very improved' end ,
```
Comment (AvrumiG) [inline TherapyClinicDB/1 Tables.sql]: You haven't added `persisted` at the end of the computation, which is recommended as per Mr. Galis

**Review by AvrumiG — CHANGES_REQUESTED** [review]
I went through your work, and it was very nicely done.
I found a few minor things to look at in the tables file and left you some comments.

P.S. Apologies for the late response—I took a break from the course last month due to moving.

File: TherapyClinicDB/1 Tables.sql (hunk 1 above)
Comment (astrohlicpu) [reply TherapyClinicDB/1 Tables.sql]: Corrected. Moved constraint to end of table.

File: TherapyClinicDB/1 Tables.sql (hunk 2 above)
Comment (astrohlicpu) [reply TherapyClinicDB/1 Tables.sql]: Fixed

File: TherapyClinicDB/1 Tables.sql (hunk 3 above)
Comment (astrohlicpu) [reply TherapyClinicDB/1 Tables.sql]: My mistake. Fixed.

**Comment by astrohlicpu** [comment]
Updated work to reflect all comments. Let me know if anything else is needed.

**Comment by AvrumiG** [comment]
All good now, great job.

**Review by AvrumiG — COMMENTED** [review]
All good now, reat job.

**Pull request closed without merging**